data/card_catalogue/social.json

```diff
@@ -99,7 +99,7 @@
         "active": "Force an NPC to act on their compulsion.",
         "passive": null
       },
-      "flavour": "He wanted to pursue the thief, but he couldn't stand leaving the candy wrappers on his spotless kitchen floor.",
```

Comment:
This one is a little esoteric. I changed it to reference the pic more directly which should hopefully make it clear what we mean.